softlayer/disk/softlayer_disk.go

```diff
@@ -0,0 +1,23 @@
+package disk
+
+import (
+	boshlog "github.com/cloudfoundry/bosh-agent/logger"
+)
+
+const softLayerDiskLogTag = "SoftLayerDisk"
+
+type SoftLayerDisk struct {
+	id     int
+	logger boshlog.Logger
+}
+
+func NewSoftLayerDisk(id int, logger boshlog.Logger) SoftLayerDisk {
+	return SoftLayerDisk{id: id, logger: logger}
+}
+
+func (s SoftLayerDisk) ID() int { return s.id }
+
+func (s SoftLayerDisk) Delete() error {
+	s.logger.Debug(softLayerDiskLogTag, "Deleting disk '%s'", s.id)
```

Comment:
So, I should not merge this? I won't until you confirm. We can then close this one. Best